src/seedu/addressbook/data/person/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private Person person;
+    private Tag tag;
+    private String status;
+    
+    private static final String TAG_ADD = "+";
+    private static final String TAG_REMOVE = "-";
+    
+    public Tagging(Person person, Tag tag, String status) {
+    	this.person = person;
+    	this.tag = tag;
+    	this.status = status;
+    }
+    
+    public String toString() {
+    	StringBuilder tagging = new StringBuilder();
+    	if(status == "ADD") {
+    		tagging.append(TAG_ADD);
+    	} else {
+    		tagging.append(TAG_REMOVE);
+    	}
+    	tagging.append(person.getName().toString());
+    	tagging.append(" ");
+    	tagging.append(tag.toString());
```

Comment:
Interestingly, if you're concatenating several well defined strings at one go (i.e. not using a loop), using a simple `String sentence = stringOne + stringTwo + stringThree` is fine. The compiler will intelligently convert that expression, and use StringBuilders to efficiently concatenate the strings.